script/infra/pybliss.py

```diff
@@ -68,6 +68,7 @@ def run_single_bliss_bench(self, args: BlissArgs) -> BlissStats:
             f"--file_type {'binary' if args.file_type else 'txt'}",
             "--use_preload" if args.use_preload else "",
         ]
+        print("this is cmd: " , " ".join(cmd))
```

Comment:
why do we have this?

Reply:
Oop sorry this was for debugging, I'll get rid of this